js/modules/markjoseph.js

```diff
@@ -0,0 +1,111 @@
+export default {
+    state: {
+        profile: {
+            username: "your-github-username",
```

Comment:
Put your GitHub username here.

```suggestion
            username: "markjoseph55",
```